enterprise/backend/src/metabase_enterprise/gsheets/api.clj

```diff
@@ -134,7 +134,7 @@
 
 (defn- is-gdrive?
```

Comment:
nit: I think we should rename this function since it's not checking if it is "a gdrive". Maybe `relevant-type?`.

Reply:
or `relevant-resource-type?`? or something else